### ISSUE TITLE
Ubuntu error # !!! BUILD FAILED --> PGP verification not OK / signature(0) verify(0)

### ISSUE THREAD

**Issue by openoms** [openoms/joininbox#102]
Getting this error on Ubuntu:
```
2022-10-31 12:42:51 (24,0 MB/s) - ‘pgp_keys.asc’ saved [5335/5335]

pub   rsa4096 2019-08-28 [SC] [expires: 2035-08-24]
      13C688DB5B9C745DE4D2E4545BFB77609B081B65
uid                      openoms <oms@tuta.io>
sub   rsa4096 2019-08-28 [E] [expires: 2035-08-24]
sub   rsa4096 2019-09-07 [S]

gpg: key 5BFB77609B081B65: "openoms <oms@tuta.io>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1

usage: git verify-tag [-v | --verbose] [--format=<format>] <tag>...

    -v, --verbose         print tag contents
    --raw                 print raw gpg status output
    --format <format>     format to use for the output

# goodSignature(0)
# correctKey(0)

# !!! BUILD FAILED --> PGP verification not OK / signature(0) verify(0)
```

Check:
```
cd /home/joinmarket/joininbox
sudo -u joinmarket git log --oneline | head -n1
```
If shows:
```
fatal: detected dubious ownership in repository at '/home/joinmarket/joininbox'
To add an exception for this directory, call:

  git config --global --add safe.directory /home/joinmarket/joininbox
```

Run:
```
git config --global --add safe.directory /home/joinmarket/joininbox

sudo bash build_joininbox.sh
```

Not sure yet what causes this on Ubuntu.

**Comment by rollomcfloogle** [comment]
I received the same error when I tried to build. I did not get this error when I ran the check:
`fatal: detected dubious ownership in repository at '/home/joinmarket/joininbox'`
However, I still ran what you suggested and the build worked. So that's good. Thanks!

**Comment by openoms** [comment]
Hope to be fixed in #109.
Please reopen if this comes up again.
